### PR TITLE
Make API Manager simplified setup compatible in both Ubuntu and Mac environments. 

### LINKED ISSUE #212: Simplified API Manager Setup doesn't work on Ubuntu
**Description:**
The simplified scripts of deploying API Manager (wso2am-ga.sh/ wso2am-latest.sh) doesn't work on Ubuntu. 
The error occurs at line 3393 (https://github.com/wso2/kubernetes-apim/blob/master/simple/single-script/wso2am-ga.sh#L3393). The single quotation after -i is creating this issue in Ubuntu (Sed command syntax is misinterpreted). Removing the single quotation will work on Ubuntu but will cause the issue on MacOs. 

**Affected Product Version:** 2.6.x 

**OS, DB, other environment details and versions:**    Ubuntu

### PR DESCRIPTION
**Pull request by NishikaDeSilva**

### CODE REVIEW

File: simple/single-script/create/funcs [hunk 1]
```diff
@@ -214,7 +214,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```
Comment (ThilinaManamgoda) [inline simple/single-script/create/funcs]: We can point this to the live site. WDYT?

File: simple/single-script/wso2am-latest.sh [hunk 2]
```diff
@@ -3435,12 +3435,12 @@ function deploy(){
     create_yaml
 
     # replace necessary variables
-    sed -i '' 's/"$ns.k8s.&.wso2.apim"/'$namespace'/g' $k8s_obj_file
-    sed -i '' 's/"$string.&.secret.auth.data"/'$secdata'/g' $k8s_obj_file
-    sed -i '' 's/"ip.node.k8s.&.wso2.apim"/'$NODE_IP'/g' $k8s_obj_file
-    sed -i '' 's/"$nodeport.k8s.&.1.wso2apim"/'$NP_1'/g' $k8s_obj_file
-    sed -i '' 's/"$nodeport.k8s.&.2.wso2apim"/'$NP_2'/g' $k8s_obj_file
-    sed -i '' 's/"$image.pull.@.wso2"/'$IMG_DEST'/g' $k8s_obj_file
+    sed -i.bak 's/"$ns.k8s.&.wso2.apim"/'$namespace'/g' $k8s_obj_file
```
Comment (ThilinaManamgoda) [inline simple/single-script/wso2am-latest.sh]: Here the Sed command creates backup files. Isn't it better to remove those? WDYT?

File: simple/single-script/create/funcs (hunk 1 above)
Comment (DilanUA) [reply simple/single-script/create/funcs]: Yes, +1, have a look on the same for EI and IS too.

File: simple/single-script/create/funcs4opensource [hunk 3]
```diff
@@ -194,7 +194,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```
Comment (DilanUA) [inline simple/single-script/create/funcs4opensource]: have a look correct this on the same for EI and IS too.

File: simple/single-script/wso2am-ga.sh [hunk 4]
```diff
@@ -3341,7 +3341,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```
Comment (DilanUA) [inline simple/single-script/wso2am-ga.sh]: have a look to correct this on the same for EI and IS too.

File: simple/single-script/wso2am-latest.sh [hunk 5]
```diff
@@ -3370,7 +3370,7 @@ function progress_bar(){
 
       if [[ $time_proc -gt 250 ]]
       then
-          echoBold "\n\nSomething went wrong! Please Follow < FAQ-Link > for more information\n"
+          echoBold "\n\nSomething went wrong! Please Follow \"https://docs.google.com/document/d/1BqRmTFu8qJG32g018mgpdgmmLuhJDDK1WgJhnL5hYe8/edit?usp=sharing\" for more information\n"
```
Comment (DilanUA) [inline simple/single-script/wso2am-latest.sh]: have a look to correct this on the same for EI and IS too.

File: simple/single-script/create/funcs (hunk 1 above)
Comment (NishikaDeSilva) [reply simple/single-script/create/funcs]: Is it hosted yet?

File: simple/single-script/wso2am-latest.sh (hunk 2 above)
Comment (NishikaDeSilva) [reply simple/single-script/wso2am-latest.sh]: Do you mean after creating use the rm command to remove the .bak file?

File: simple/single-script/create/funcs (hunk 1 above)
Comment (ThilinaManamgoda) [reply simple/single-script/create/funcs]: It should be hosted here https://wso2.com/products/install/faq/#Kubernetes. We put Simplified deployment related info to the FAQ  but it seems like it is not in the live yet.

File: simple/single-script/wso2am-latest.sh (hunk 2 above)
Comment (ThilinaManamgoda) [reply simple/single-script/wso2am-latest.sh]: yes

File: simple/single-script/create/funcs (hunk 1 above)
Comment (ThilinaManamgoda) [reply simple/single-script/create/funcs]: Now it's live https://wso2.com/products/install/faq/#Kubernetes

**Pull request merged**